frontend/endpoints/destination_recognition/destination_finder.go

```diff
@@ -2,16 +2,20 @@ package destination_recognition
 
 import (
 	"github.com/gin-gonic/gin"
+	"github.com/odigos-io/odigos/frontend/kube"
+	"github.com/odigos-io/odigos/k8sutils/pkg/client"
 	k8s "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 type DestinationType string
 
 const (
-	JaegerDestinationType DestinationType = "jaeger"
+	JaegerDestinationType        DestinationType = "jaeger"
```

Comment:
We already have these defined in https://github.com/yodigos/odigos/blob/main/common/dests.go